conda/recipes/libcudf/recipe.yaml

```diff
@@ -28,6 +28,9 @@ cache:
         export CXXFLAGS=$(echo $CXXFLAGS | sed -E 's@\-fdebug\-prefix\-map[^ ]*@@g')
         set +x
 
+        cudf_ROOT="$(realpath ./cpp/build)"
```

Comment:
Maybe add a comment here -- but also if this is temporary and will be replaced by a longer-term fix, don't worry about it.